components/project-list/project-list.jsx

```diff
@@ -29,14 +29,11 @@ export default React.createClass({
   fetchData(params = {}) {
     Service.entries
       .get(params)
-      .then((entries) => {
-        // sort data from mostly -> least recently added
-        entries = entries.sort((a,b) => {
-          return b.id - a.id;
-        });
+      .then((response) => {
+
         this.setState({
           loadedFromGoogle: true,
-          entries: entries
+          entries: response
```

Comment:
btw ES6 allows for a shorter object notation if the property (in-scope) has the same name as the value, so you could do:
```
.then(entries => this.setState({ loadedFromGoogle: true, entries });
```
And done: the `entries` part is treated as if you wrote `entries: entries`